js/bitmex.js

```diff
@@ -1180,6 +1180,22 @@ module.exports = class bitmex extends Exchange {
         return false;
     }
 
+    async fetchStatus (params = {}) {
+        try {
+            const { timestamp } = await this.request('');
```

Comment:
Unfortunately, we can't transpile destructuring assignments.
Also, this method does not really check the status of the exchange and in case of any other error it will report that the exchange is on maintenance, which is technically an error. I suggest that we don't invent this method for BitMEX, if they don't have a standalone endpoint for the exchange status. What do you think?

Reply:
Yep, unfortunately BitMEX doesn't have standalone endpoint to check update/maintenance/active status 😞 But I can check http status (code, message, etc) when BitMEX will go down on maintenance next time. Maybe it will be not only 503 code without any JSON-response, so we could use it to detect real status
Deal?)

Reply:
> But I can check http status (code, message, etc) when BitMEX will go down on maintenance next time.

If that happens occasionally, we already handle this in `handleErrors`, so in this case I would not merge `fetchStatus`, because it really hides other errors.

Reply:
I see) Thanks for answers!
I agree - this PR is needless

Reply:
@alexeymarunin ok, closing this for now. We will be happy to get more PRs from you in the future! Looking forward! Thank you!